tests/test_mappings/test_imports/test_modules/test_base.py

```
@@ -413,3 +414,55 @@
+    sage_intacct_connection = SageIntacctConnector(credentials_object=intacct_credentials, workspace_id=workspace_id)
+
+    sage_intacct_connection.sync_projects()
+
+    new_projects = DestinationAttribute.objects.filter(workspace_id=workspace_id, attribute_type='PROJECT').count()
+    assert new_projects == 13
+
+    new_projects = DestinationAttribute.objects.filter(workspace_id=workspace_id, attribute_type='PROJECT', active=True).count()
+    assert new_projects == 13
+
+    project = DestinationAttribute.objects.filter(workspace_id=workspace_id, attribute_type='PROJECT', value='Mobile App Redesign').first()
+    assert project.active == True
```

Ruff

Avoid equality comparisons to `True`; use `if project.active:` for truth checks

Replace with `project.active`

(E712)